pym/_emerge/depgraph.py

```diff
@@ -885,10 +880,70 @@ def _show_ignored_binaries(self):
 					self._dynamic_config.ignored_binaries.pop(pkg)
 					break
 
-			else:
-				for reason, info in self._dynamic_config.\
-					ignored_binaries[pkg].items():
-					ignored_binaries.setdefault(reason, {})[pkg] = info
+	def _ignored_binaries_autounmask_backtrack(self):
+		"""
+		Check if there are ignored binaries that would have been
+		accepted with the current autounmask USE changes.
+
+		@rtype: bool
+		@return: True if there are unnecessary rebuilds that
+			can be avoided by backtracking
+		"""
+		if not self._dynamic_config._allow_backtracking:
+			return False
+
+		if not self._dynamic_config._needed_use_config_changes:
+			return False
+
+		if not self._dynamic_config.ignored_binaries:
+			return False
```

Comment:
are these already evaluated? not properties...  Then this could be shortened to:

if False in [ self._dynamic_config._allow_backtracking,
                  self._dynamic_config._needed_use_config_changes,
 		  self._dynamic_config.ignored_binaries]:
			return False